packages/realm-server/tests/realm-server-test.ts

```diff
@@ -2063,7 +2054,8 @@ module('Realm Server', function (hooks) {
 
     module('permissioned realm', function (hooks) {
       setupPermissionedRealm(hooks, {
-        john: ['read'],
+        john: ['read', 'write', 'realm-owner'],
+        'test-worker': ['read', 'write', 'realm-owner'],
```

Comment:
we need to follow the same pattern in our tests that we do in real life. so any realm bots should start with `realm/` in the username